src/Microsoft.AspNet.Mvc.ModelBinding/Binders/FormCollectionModelBinder.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Http;
```

Comment:
base `Controller` class uses this namespace.  but how would users discover `FormCollection`?  that's in `Microsoft.AspNet.Http.Core.Collections`